.github/workflows/scorecard.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       # Upload the results as artifacts (optional). Commenting out will disable uploads of run results in SARIF
       # format to the repository Actions tab.
       - name: "Upload artifact"
-        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
+        uses: actions/upload-artifact@v4.6.0 # v4.6.0
```

Comment:
@insaaniManav Can we please find this to commit SHA? I know its painful but it just mitigates a risk as well as improve our score in OpenSSF scorecard